features/example/example.md

```diff
@@ -0,0 +1,25 @@
+# Example feature spec
+This contains an example of a new feature to be added.
+
+## Overview
+The purpose of this new feature is to be an example. It will modify the
+behavior of Nimbess in the following ways...
+
+## Functional Spec
+Indicate how your change will function. Describe in detail how it will work.
+
+## Code Changes
+Include a summary of the code changes necessary to implement your new
+functionality.
+
+## API/Schema changes
+Include any changes to the current API as well as database schema changes.
+
+## Performance Impact
+List any concerns around performance increase/decrease with your change.
+
+## User Impact
+List any concerns around user experience with your change.
+
+## Owners
+List all owners that will collaborate to implement this feature.
```

Comment:
Add a test section to ensure we have tests?

Reply:
How do you want that section to look? Contributor provides an overview of the test cases or areas that will need testing?

Reply:
I added a section requesting an overview of testing required.